draft-ietf-taps-impl.md

```diff
@@ -963,3 +963,46 @@ If the application indicates a bound on the expected Send or Receive bitrate, an
 
 * Cost Preferences:
 If the application indicates a preference to avoid expensive paths, and some paths are associated with a monetary cost, an implementation should decrease the ranking of such paths. If the application indicates that it prohibits using expensive paths, paths that are associated with a cost should be purged from the decision tree.
+
+# Reasons for errors {#appendix-reasons-errors}
+
+The Transport Services API {{I-D.ietf-taps-interface}} allows for the seven generic error types to specify a more detailed reason as to why an error occurred. This appendix lists some of the possible reasons.
```

Comment:
I'm not sure we really need to count the error types here, so I suggest deleting "the seven" here.

Reply:
Agreed. Suggest using "several" or "various".